chapter_1/1_is_unique/lib/is_unique.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Implement an algorithm to determine if
+# a string has all unique characters. What if you
+# cannot use additional data structures?
+
+class IsUnique
+  def check_unique(string)
+    letters = string.downcase.scan(/[a-z]/)
```

Comment:
Just out of curiosity is there a reason why you specifically used `scan` vs `split`?

Reply:
no particular reason, I hadn't come across scan before so I decided to try it out instead of split